packages/core/src/test/testUtil.ts

```diff
@@ -614,3 +614,25 @@ export function tryRegister(command: DeclaredCommand<() => Promise<any>>) {
 export function getFetchStubWithResponse(response: Partial<Response>) {
     return stub(request, 'fetch').returns({ response: new Promise((res, _) => res(response)) } as any)
 }
+
+export function setEnvPath(newPath: string): void {
+    process.env.PATH = newPath
+}
+
+export function readEnvPath(): string {
+    return process.env.PATH || ''
+}
+/**
+ * Overwrite the $PATH environment variable for the span of the provided task, then reset it.
+ * @param newPath temporary $PATH value
+ * @param task work to be done with $PATH set.
+ */
+export async function withEnvPath(newPath: string, task: () => Promise<void>): Promise<void | never> {
```

Comment:
not a blocker, but future idea: this could be generalized to `withEnv()` or even just `with()`:

```
with({
  env: ...
})
```